backend/fetchStopsByPostcode.ts

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios';
+import * as dfns from 'date-fns';
+import { Bus, type BusArrayDto, getArrivals } from './fetchArrivals';
+
+
+//take a postcode and return a long lat NaptanBusCoachStation 
+async function getLatLongByPostcode (postcode : string){
+    let url = "https://api.postcodes.io/postcodes";
+
+    postcode = postcode.replace(' ','');
+    let result: [boolean, number, number] = [false, 0, 0];
+
+    if (postcode !== undefined || postcode !== "" ){
+        try{
+            url += `/${postcode}`;
```

Comment:
why do we append postcode here in the try catch? is there a reason not to create the url in one go using string interpolation?